librz/core/cmd/cmd_analysis.c

```diff
@@ -1556,6 +1556,7 @@ static void cmd_analysis_esil(RzCore *core, const char *input) {
 	case 'A': // "aeA"
 		if (input[1] == '?') {
 			rz_core_cmd_help(core, help_msg_aea);
+			rz_cons_printf("");
```

Comment:
why this?

Reply:
Oops, really sorry about this, I should be much much more careful in cleaning up my code before pushing commits to remote branches. My bad